Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm

```diff
@@ -96,8 +96,10 @@ - (void)authorizationController:(ASAuthorizationController *)controller didCompl
 namespace WebKit {
 using namespace WebCore;
 
-static inline Ref<ArrayBuffer> toArrayBuffer(NSData *data)
+static inline RefPtr<ArrayBuffer> toArrayBuffer(NSData *data)
 {
+    if (!data || !data.length)
+        return nullptr;
```

Comment:
It's not very obvious how this change fixes the issue you describe in the ChangeLog, because this is such a generically named method. It might be slightly clearer if the location where `toArrayBuffer` was called did the null check, but I think this is fine (especially if it's used in multiple places).

Reply:
How come there is no test?